### ISSUE TITLE
Options not saved

### ISSUE THREAD

**Issue by DLongoni** [rncbc/qtractor#127]
Hello, 
this issue occurs for me under ubuntu 16.04, after installing qtractor 0.8.6 via "apt-get", and also after compiling from source both versions 0.8.5 and 0.8.6.

this are the steps to take to reproduce the issue:

- open qtractor (via terminal or via launcher)
- change any option (example: maximum number of recent files)
- close qtactor
- reopen qtractor, the options keep no trace of the changes made

If I then open qtractor from terminal as super user and change any option, then the changes are saved and I see them properly changed also opening qtractor without sudo.

The options configuration file is in ~/.config/rncbc.org/Qtractor.conf
Since I thought it could be a permission issue, I changed the file permission to 777 just to try, and nothing changed.
I tried even chowning it to the user running qtractor, but again nothing changed.

Thanks for your help

**Comment by rncbc** [comment]
that's not reproduced here:
of course, changing any options do have their due effects thereafter, as expected by design.
something is not right on your side, sorry to tell.
please be a little more investigative and every time you change an option, check whether it makes it to the ~/.config/rncbc.org/Qtractor.conf file or else...
hope you sort it out
cheers

**Comment by DLongoni** [comment]
Firstly, thanks for the reply. 
I agree that there is something wrong on my side, I was hoping I could find some insight on how to solve the issue.
I can tell you for instance that the same behaviour occurs for audio metronome options, and evenfor recent files. That is, if I open a file, then close and reopen qtractor then I won't see it listed under recents. On the other hand, if I sudo qtractor and open a file, then it will be listed under recent if I both open it as a regular user or as a super user. 

I believe the place to look at is the QSettings constructor in qtractorOptions.cpp:
qtractorOptions::qtractorOptions (void)
	: m_settings(QTRACTOR_DOMAIN, QTRACTOR_TITLE)

In particular refering to QSettings doc 
http://doc.qt.io/qt-5/qsettings.html
in the "Fallback Mechanism" section. The system scope vs user scope mechanism is explained.

On the one hand I would think that QSettings on my system somehow can't find the right config file to be saved. On the other hand the right config file is loaded, so it does not make sense to me that QSettings is loading from one file and saving to another. I would exclude also that it's a permission problem since I already changed the config file access permission.

Does this additional analysis ring any bell for you?

Thanks again.

**Comment by rncbc** [comment]
please check on the [Platform-Specific Notes](http://doc.qt.io/qt-5/qsettings.html#platform-specific-notes) whether you have a _stumbler_ on  the `$XDG_CONFIG_DIRS` ...`/rncbc.org/Qtractor.conf`

**Comment by DLongoni** [comment]
My aim is just to find a solution to my problem, I have clearly no interest in changing the code for my own sake. And I trust you, don't worry (=

As a matter of fact I just checked all the possible locations including $XDG_CONFIG_DIRS subdirectories, but the only qtractor.conf to be found was ~/.config/rncbc.org/Qtractor.conf

I had already tried to remove it and have qtractor regenerate it, with no success. 

I finally solved the issue removing the whole rncbc.org folder. Restarting qtractor recreated the folder and Qtractor.conf inside it. The changes then were persistent.

Thank you for you help.
